iPetCare/API/Controllers/PetsController.cs

```diff
@@ -19,13 +21,33 @@ public PetsController(IPetService petService)
         }
 
         [Produces(typeof(ServiceResponse<PetsGetPetsDtoResponse>))]
+        [AuthorizeRoles(Roles = Role.Administrator)]
         [HttpGet]
+        [AuthorizeRoles(Roles = Role.Administrator)]
```

Comment:
2 razy authroize roles? Czy sie cos na gicie popsuło?

Reply:
2 razy, to u mnie coś w glowie się popsuło xD